### DISCUSSION TITLE
Search Bar

### DISCUSSION THREAD

**Discussion by mkitzmann** [GoPlan-Finance/goplan-app#65] [Ideas]
I suggest implementing a search bar in the header which should look like this:

![image](https://user-images.githubusercontent.com/35574021/111042155-38c0dd80-843c-11eb-9ca4-62e09666db3d.png)

It should show the assets that match the input string and load the necessary data from an api endpoint on input. 
The input should be debounced to prevent too many api calls.
When one of the items is clicked, the details page for this asset should be shown.

We could use one of these external api:
https://www.alphavantage.co/documentation/#symbolsearch
https://api.tiingo.com/documentation/utilities/search

**Comment by sadortun** [comment]
If a user search something and can't find a result. Add a link to a GH discussion where he can report what's missing.

We could also log searches, but this might create a lot of low quality data...
